Task2/NumberParser.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Task2
```

Comment:
Please, format document before push
 To format a document: Ctrl+K, Ctrl+D in VisualStudio

Reply:
I did it but I didnt see any difference